### PR TITLE
NS-314 Avoid duplicates in sis_dropped_classes table

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/NS-314

### CODE REVIEW

File: nessie/sql_templates/create_intermediate_schema.template.sql [hunk 1]
```diff
@@ -81,6 +81,10 @@ AS (
             en.enrollment_status = 'D'
             OR en.grade = 'W'
         )
+    /* Clear out duplicates. */
+    GROUP BY
```
Comment (sandeepmjay) [inline nessie/sql_templates/create_intermediate_schema.template.sql]: @raydavis - Would we need a having clause?

Comment (raydavis) [reply nessie/sql_templates/create_intermediate_schema.template.sql]: Not that I can see off-hand, @sandeepmjay -- please say more!

Comment (sandeepmjay) [reply nessie/sql_templates/create_intermediate_schema.template.sql]: Looks like this was a filtering query. Thanks for the explanation @raydavis.

**Comment by sandeepmjay** [comment]
LGTM @raydavis.

**Pull request merged**